Source/WebCore/rendering/svg/SVGTextMetricsBuilder.h

```diff
@@ -57,6 +58,7 @@ class SVGTextMetricsBuilder {
 
     // Simple text only.
     std::unique_ptr<WidthIterator> m_simpleWidthIterator;
+    std::unique_ptr<ComplexTextController> m_complexTextController;
```

Comment:
seems like this should not be in the `// Simple text only.` section but should instead be in the `// Complex text only.` section below